src/components/menu/__tests__/test_menubarButton.tsx

```
@@ -0,0 +1,64 @@
+import * as React from "react";
+import Menubar from "../component";
+import renderer from "react-test-renderer";
+
+describe("Menubar", () => {
+    it("renders closed menubar correctly", () => {
+        const tree = renderer
+            .create(
+                <Menubar isOpen={false} onClose={() => {}}>
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)